src/Tools/RepoUtil/ChangeCommand.cs

```diff
@@ -154,6 +154,13 @@ private List<NuGetPackageChange> CalculateChanges(IEnumerable<NuGetPackage> pack
             var map = _repoData
                 .FloatingPackages
                 .ToDictionary(x => x.Name, Constants.NugetPackageNameComparer);
+            foreach (var @fixed in _repoData.FixedPackages)
+            {
+                if (@fixed.GenerateNameOpt != null)
+                {
+                    map.Add(@fixed.GenerateNameOpt, @fixed.Package);
+                }
+            }
```

Comment:
Woops, I was reading the wrong line. You're right, I need to test that feature more.